### ISSUE TITLE
CUDA  6

### ISSUE THREAD

**Issue by red-riv** [xmrig/xmrig-cuda#115]
How to mine on a CUDA 6 GPU?

**Comment by Spudz76** [comment]
You don't.

CUDA 8.0 minimum, with Compute Capability 2.0 or higher (Fermi chip or better).

Note Techpowerup site shows "CUDA Version" which is actually Compute Capability (or "arch") and has nothing to do with any versions of CUDA Toolkit...  So if you mean a Pascal-based card then you just get CUDA Toolkit 11.4 and the latest drivers and it works (Techpowerup saying "version" 6.0 means *architecture* 60 or Compute Capability 6.0, which is a Pascal type).  You can use `-DCUDA_ARCH=60` to save compilation time if this is the only GPU (normal build makes code for ALL types).

**Comment by red-riv** [comment]
So if I install CUDA Toolkit 11.4, will I be able to mine using my MX110 and GT 1030?

**Comment by Spudz76** [comment]
Yes but then either omit `-DCUDA_ARCH` so it builds everything, or use `-DCUDA_ARCH=50;61` to build just the kernels for MX110 (5.0) and GT1030 (6.1).

One caveat, "some" laptop Maxwell chips were dropped from CUDA so if yours is one of those (there is no list) you might have to use Toolkit 10.2 maximum.  Or if your driver is older, check `nvidia-smi` command for which CUDA the driver supports and use that Toolkit to match (newer Toolkit than driver won't work).
